libs/ui-watt/src/lib/components/table/watt-table.component.ts

```diff
@@ -210,6 +210,12 @@ export class WattTableComponent<T>
   @Input()
   selectable = false;
 
+  /**
+   * Sets the initially selected rows. Only works when selectable is `true`.
+   */
+  @Input()
+  initiallySelectedRows: T[] = [];
```

Comment:
suggestion: how about `initialSelection` instead? seems more in line with `selectionChange` and how variables for initial values usually are named.